### PR TITLE
Migrating from Akka to Pekko

### PR DESCRIPTION
**Pull request by ncreep**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -35,13 +30,13 @@ val commonsCodec = "commons-codec" % "commons-codec" % "1.15"
 val bouncyCastle = "org.bouncycastle" % "bcpkix-jdk18on" % "1.76"
 
 
-// the client API request/response handing uses Akka Http
-val akkaHttp = ("com.typesafe.akka" %% "akka-http" % "10.2.9").cross(CrossVersion.for3Use2_13)
-val akkaStream = ("com.typesafe.akka" %% "akka-stream" % akkaVersion).cross(CrossVersion.for3Use2_13)
-val akka = ("com.typesafe.akka" %% "akka-actor" % akkaVersion).cross(CrossVersion.for3Use2_13)
+// the client API request/response handing uses Pkka Http
```
Comment (hagay3) [inline build.sbt]: Pkka -> Pekko

File: examples/src/main/scala/skuber/examples/guestbook/README.md [hunk 2]
```diff
@@ -110,7 +110,7 @@ You can verify the Guestbook application is available by navigating to a service
 
 ## Design
 
-The design of the reactive guestbook example is based on the [actor model](https://en.wikipedia.org/wiki/Actor_model) and uses [Akka](http://doc.akka.io/docs/akka/2.4.1/intro/what-is-akka.html). Note that Skuber itself has no dependency on Akka - an actor model was chosen for this example simply because it seemed an appropriate abstraction, especially due to the asynchronous and reactive nature of the processing. The actors in this example run purely locally.
+The design of the reactive guestbook example is based on the [actor model](https://en.wikipedia.org/wiki/Actor_model) and uses [Pekko](https://pekko.apache.org/what-is-pekko.htmll). Note that Skuber itself has no dependency on Pekko - an actor model was chosen for this example simply because it seemed an appropriate abstraction, especially due to the asynchronous and reactive nature of the processing. The actors in this example run purely locally.
```
Comment (hagay3) [inline examples/src/main/scala/skuber/examples/guestbook/README.md]: `htmll` -> `html`

**Review by hagay3 — APPROVED** [review]

File: build.sbt (hunk 1 above)
Comment (ncreep) [reply build.sbt]: A nice hybrid of Pekko and Akka.

File: examples/src/main/scala/skuber/examples/guestbook/README.md (hunk 2 above)
Comment (ncreep) [reply examples/src/main/scala/skuber/examples/guestbook/README.md]: :heavy_check_mark:

**Pull request merged**